app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
```

Comment:
- [ ] According to the project requirement, The application allows for registering a new user. However, I'm not able to create a new User 🤔 Please fix it!! 

![imaThe application allows for registering a new user.ge](https://github.com/fickryiman/RubyOnRails-Blog/assets/79058364/18b9dd16-0222-49af-b95e-fab69fa2352b)

- [ ] Also, make sure A user can correct and reset the password in the application. ✍🏻